bin/package_build.py

```diff
@@ -332,6 +332,42 @@ def getIBMValidatedOpenSourceList(oskey):
 			print("Couldn't fetch appropriate package for given command.")
 
 
+def zOSOpenTools():
+    url = "https://api.github.com/users/ZOSOpenTools/repos"
+    repos = []
+    page = 1
+    per_page = 100
+
+    while True:
+        params = {'page': page, 'per_page': per_page}
+        response = requests.get(url, params=params)
+        
+        if response.status_code == 200:
+            repos.extend(response.json())
+            if 'next' in response.links:
+                page += 1
+            else:
+                break
+        else:
+            print("Failed to fetch repositories:", response.status_code)
+            break
+    
+    zOS_list = []
+    for repo in repos:
+        repo_info = {"packageName": repo["name"], "description": repo["description"], "version": ""}
+        zOS_list.append(repo_info)
+
+    directory = './distro_data/'
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+    file_path = os.path.join(directory, 'ZOSOpenTools.json')
+
+    with open(file_path, 'w') as file:
+        json.dump(zOS_list, file, indent=2)
```

Comment:
My suggestion regarding the logic above is to look at how the other functions save the files.

Specifically, there's output so the user knows the file has saved:

print(f"Saved!\nfilename: {q}")

So you'll want to review the code a bit to make sure something like that works, and some of this could be simplified a bit based on other examples in the script.